docs/articles/documentation/guides/advanced-guides/test-https-features-and-http2-websites.md

```diff
@@ -23,21 +25,107 @@ If the tested page does not use HTTPS-specific features (like
 [Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API),
 [Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API),
 [ApplePaySession](https://developer.apple.com/documentation/apple_pay_on_the_web/applepaysession), or
-[SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto)), TestCafe successfully proxies it over HTTP. Otherwise, the tests fail because the page throws JavaScript errors, does not load or loads partially. This indicates that you should [configure the proxy server to use HTTPS](#proxy-the-website-over-https).
+[SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto)), TestCafe successfully proxies it over HTTP. Otherwise, the tests fail because the page throws JavaScript errors, does not load or loads partially.
```

Comment:
"[SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto), TestCafe successfully proxies it over HTTP" This does not make sense.

Reply:
GitHub cut it because of line breaks. Full sentence goes:

"If the tested page does not use HTTPS-specific features (like Service Workers, Geolocation API, ApplePaySession, or
SubtleCrypto), TestCafe successfully proxies it over HTTP."

Makes sense?

Reply:
...TestCafe can proxy it over HTTP.